android-compose/koin-androidx-compose/src/main/java/org/koin/androidx/compose/ComposeExt.kt

```diff
@@ -17,30 +17,33 @@ package org.koin.androidx.compose
 
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.remember
-import org.koin.core.Koin
-import org.koin.core.context.GlobalContext
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.KoinScopeComponent
+import org.koin.core.component.get
 import org.koin.core.parameter.ParametersDefinition
 import org.koin.core.qualifier.Qualifier
 
-/**
- * Resolve a dependency for [Composable] functions
- * @param qualifier
- * @param parameters
- *
- * @return Lazy instance of type T
- *
- * @author Arnaud Giuliani
- * @author Henrique Horbovyi
- */
 @Composable
-inline fun <reified T> get(
```

Comment:
Why replace the `get` function? Either we can deprecate it or keep it 🤔

Reply:
I only changed its place in code and removed the docs (by accident). It's here https://github.com/InsertKoinIO/koin/pull/1178/files/6062bdfdb3b2fc25f38acb26a0b8995b9480e508#diff-f9db5f6b8a1cbf8a2b9601a97856b228a48df598a9fb60c8e451a43b8ce5ccd2R41